core/src/main/java/com/netflix/conductor/metrics/Monitors.java

```diff
@@ -267,4 +267,16 @@ public static void recordDaoError(String dao, String action) {
 	public static void recordAckTaskError(String taskType) {
 		counter(classQualifier, "task_ack_error", "taskType", taskType);
 	}
+
+	public static void recordESIndexTime(String docType, long val) {
+		getTimer(Monitors.classQualifier, docType, docType).record(val, TimeUnit.MILLISECONDS);
```

Comment:
I would like to ask two questions here,
1) We are using prometheus metrics. I see timer values as ```percentile="T0072",statistic="percentile"```. How to calculate 90 percentile value. As of now We are using totalTime / count to get average value.
2) How does  ```statistic="totalTime"``` calculated. Let say some task execution took 429ms. I see value as 4.29E9 although time unit says it is in seconds.
@apanicker-nflx @kishorebanala

Reply:
@manan164 I'm not very familiar with Prometheus. We use [Atlas](https://github.com/Netflix/atlas) for collecting metrics.